_plugins/brooklyn_metadata.rb

```diff
@@ -44,7 +44,7 @@ def generate(site)
         }
       end
       
-      url_set["git"] = "https://github.com/apache/brooklyn/tree/#{ git_branch }"
+      url_set["git"] = "https://github.com/apache/brooklyn-server/tree/#{ git_branch }"
```

Comment:
Yes, it needs re-working. For this though, I manually checked all the places this is being used and they're all currently in brooklyn-server.